### PR TITLE
Add a config option to allow multiple taverns

### PR DESCRIPTION
**Pull request by ToMe25**
# Changes proposed in this pull request:
- Add a config option to allow multiple taverns in one colony
- as suggested in ldtteam/minecolonies-features#144
-

Review please

### CODE REVIEW

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 1]
```diff
@@ -74,6 +74,8 @@
   "minecolonies.config.builderinfiniteresources.comment": "Should the Builder and Miner build without resources? (This also turns off what they produce.) [Default: false]",
   "minecolonies.config.limittoonewarehousepercolony": "Warehouse Limit",
   "minecolonies.config.limittoonewarehousepercolony.comment": "Should there be a max of 1 warehouse per colony?. [Default: true]",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "minecolonies.config.limittoonewarehousepercolony.comment": "Should there be a max of 1 warehouse per colony? [Default: true]",
```

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 2]
```diff
@@ -74,6 +74,8 @@
...
+  "minecolonies.config.limittoonetavernpercolony": "Tavern Limit",
+  "minecolonies.config.limittoonetavernpercolony.comment": "Should there be a max of 1 Tavern per colony?. [Default: true]",
```
Comment (ravenbuilder934) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
```suggestion
  "minecolonies.config.limittoonetavernpercolony.comment": "Should there be a max of 1 Tavern per colony? [Default: true]",
```

Comment (ToMe25) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: I didn't read it, i just copy pasted it xD

**Comment by Raycoms** [comment]
to be fair, not sure if we want that. I mean having multiple warehouses makes sense to some degree and it's a config value because it's currently not really stable. But multiple taverns?

File: src/main/resources/assets/minecolonies/lang/en_us.json (hunk 2 above)
Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: My fault for not seeing the period in my language PR 😄

**Comment by ravenbuilder934** [comment]
London alone has over 3,000 taverns/pubs. This would be more for late-game colonies, so a colonist doesn't have to walk 10-15 chunks to get a drink. 😛

**Comment by ToMe25** [comment]
Tbf its kinda useless because you can't get new citizens as long as the only free beds are in taverns,
there just wasn't any comment on the issue, and also i though it would be nice as decoration so i made it.

**Comment by someaddons** [comment]
the features are a gathering of suggestions, if you want to add one you should talk with us guys before. The tavern is currently intentionally designed about beeing 1 per colony, so no we do not want to add this

**Comment by ToMe25** [comment]
I will close this then, should i close the issue too?

**Comment by someaddons** [comment]
leave the issue open, we got a label for won't be added. Just so that feature suggestions of the same are stacked up

**Comment by ToMe25** [comment]
ok, will add that to the issue then.

**Pull request closed without merging**